CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,175 @@
+# pyOpenSci Community Code of Conduct
+
+pyOpensci is dedicated to creating a welcoming, supportive and diverse 
+community around the open source Python tools that drive open
+science. We do this through a combination of community building, 
+mentorship, peer review of software and education and outreach. 
+
+We are building the pyOpenSci community in a way that promotes fairness 
+and justice for all, with a particular focus on supporting those from 
+traditionally marginalized / underrepresented groups. We want every member of 
+our community to thrive; we want to make a positive impact on our community 
+members' work and on the work of others. 
+
+### What this code of conduct defines 
+
+Our Code of Conduct defines expected behavior 
+and guidelines that help create such a community.
+
+Anyone who participates in a pyOpenSci activity either online or in person is expected to show respect and courtesy to others in all interactions. These interactions may occur in a variety of locations including:
+
+* in GitHub repositories
+* in our Slack channel
+* in our Discourse online forum 
+* during in-person events, 
+* when representing pyOpenSci in public, 
+* or in any event online or in person where pyOpenSci is leading the event.  
+
+To make sure that everyone has a common understanding of “show respect and courtesy to each other,” we have adopted the following code of conduct. The code of conduct is enforced by the pyOpenSci Executive Director and the Advisory Council.
+
```

Comment:
ED and Advisory Council -> CoC Stewards

Reply:
i changed the language to this:

The code of conduct is enforced by the pyOpenSci Executive Director and the code of conduct stewardship committee.

which will allow me to ask my new hire to be on the stewardship committee and then one other person?